app/src/androidTest/kotlin/com/google/samples/apps/nowinandroid/ui/NavigationTest.kt

```diff
@@ -267,4 +267,28 @@ class NavigationTest {
             onNodeWithTag("topic:${topic.id}").assertExists()
         }
     }
+
+    @Test
+    fun navigatingToTopicFromForYou_showsTopicDetails() {
+        composeTestRule.apply {
+            // Follow a topic
+            onNodeWithText(sampleTopic).performClick()
+
+            // Get the topic ID
+            val topic = runBlocking {
+                topicsRepository.getTopics().first().filter { it.name == sampleTopic }.first()
+            }
+
+            // Tap the first topic chip
+            onNodeWithTag("topicChip:${topic.id}", useUnmergedTree = true)
+                .assertExists()
```

Comment:
```
java.lang.AssertionError: Failed: assertExists.
Reason: Expected exactly '1' node but found '2' nodes that satisfy: (TestTag = 'topicChip:1')
Nodes found:
1) Node #209 at (l=315.0, t=2707.0, r=516.0, b=2833.0)px, Tag: 'topicChip:1'
Has 1 child
2) Node #239 at (l=1293.0, t=2896.0, r=1494.0, b=3022.0)px, Tag: 'topicChip:1'
Has 1 child, 2 siblings
```

Reply:
@JoseAlcerreca Would love to chat about this when you have a minute. tl;dr adding UI tests is _incredibly_ painful, think I must be doing something wrong.